### PR TITLE
Issue19 upgrade to angular6

### PR DESCRIPTION
**Pull request by israelramosm**

### CODE REVIEW

**Comment by geoperez** [comment]
Instead of using a HttpOptions, try to create an interceptor to receive the request before to send (https://angular.io/guide/http#intercepting-requests-and-responses)

@kadosh what do you think??

**Comment by kadosh** [comment]
The interceptor is the way to go in this case. You can also take a look at TB1 where an interceptor is already in place.

**Comment by geoperez** [comment]
Go team go

File: projects/tubular-lib/src/lib/interceptors/interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpInterceptor, HttpHandler, HttpRequest, HttpResponse, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { GridResponse } from '../grid/grid-response';
+import { tap, finalize } from 'rxjs/operators';
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
```
Comment (geoperez) [inline projects/tubular-lib/src/lib/interceptors/interceptor.ts]: Rename the file to match the class name.

**Review by geoperez — CHANGES_REQUESTED** [review]

**Review by geoperez — APPROVED** [review]

**Pull request merged**